exercises/sublist/sublist.py

```diff
@@ -1,2 +1,2 @@
-def check_lists():
+def check_lists(firstList, secondList):
```

Comment:
Python favors `snake_case` over `CamelCase`. 
A good Python style guide from Google - https://google.github.io/styleguide/pyguide.html?showone=Naming#Naming

Could you please change it?

Reply:
Sorry for this, I just started to using Python and I've never seen this guide before. So i just have to change ```firstList``` to ```first_list``` and ```secondList``` to ```second_list```. Thanks for the guide.

Reply:
@cruxiu you're welcome